Assets/Scripts/Explosion.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Explosion : MonoBehaviour {
+
```

Comment:
this extra new line should be removed once you have the bracket on the line below